src/dataframes.jl

```diff
@@ -87,7 +87,7 @@ function _get_col_subset(X, cols::Vector{Symbol}; return_vector=false)
     end
 
     try
-        return convert(Array, return_vector ? X[:, cols[1]] : X[:, cols])
+        return Array(return_vector ? X[:, cols[1]] : X[:, cols])
```

Comment:
Yes. Constructors always worked. Just `convert` is a legacy think from pre-Julia 1.0 that we kept for some time to avoid breaking changes (and we have these breakages when we removed them 😄).